README.md

```diff
@@ -1,104 +1,122 @@
-# PAC-ID Resolver (DRAFT)
+# PAC-ID Resolver
 
-## Introduction
-In laboratory workflows, it is often necessary to utilize multiple software applications and devices. Dealing with chemicals, samples, and instruments requires a seamless exchange of contextual data between these applications and devices, significantly benefiting scientists and lab technicians.
+## `PAC-ID Resolver` in a Nutshell
 
-The conventional approach involves interfacing applications and devices with each other. Unfortunately, due to the lack of standardized support, such interfaces usually require bespoke software development or customization projects, demanding substantial effort to create and maintain.
+A `PAC-ID Resolver`, short for **P**ublicly **A**ddressable **C**ontent **ID**entifier **Resolver**, is an architectural pattern for loosely coupling applications, creating modern user experience without classical integration. The basis for the coupling is a `PAC-ID`. 
 
-The PAC-ID resolver architecture introduced here offers an alternative and lightweight solution for implementing handover between applications and devices in a loosely coupled manner, providing an exceptional user experience.
+Working principle: The application asks the `PAC-ID Resolver` to convert a given `PAC-ID` into an ordered list of coupling information, based on configurable coupling information tables provided to the `PAC-ID Resolver`. The application in turn selects the best coupling information from the list, based on an application intent. The coupling information contains a URL, which can be used by the application to either handover the user to another application or to find relevant end-points for inter application communication.
```

Comment:
The coupling information, depending on the indicated service type, contains a URL (e.g. a deep link to a target system) for user handovers or a endpoint for standardized inter-application communication.

(The original wording indicates that the URL is for finding relevant endpoints but in reality the url itself is the endpoint.